src/ui/components/markdown-viewer/component.ts

```diff
@@ -1,5 +1,12 @@
 import Component from '@glimmer/component';
 
+interface Window {
+  hljs: any;
+  markdownit: any;
+}
+
+declare var window: Window;
```

Comment:
I don't think we want to do all this. It redefines the type for `window`. For `hljs`, I think we can just remove `window.` as it is already declared as a namespace in `node_modules/@types/...`. For `markdownit` you could remove `window.` as well and `declare const markdownit: (options: object) => any;` in `utils/types.d.ts`.